package.json

```diff
@@ -118,6 +118,7 @@
         "@angular/cli": "17.3.7",
         "@angular/compiler-cli": "17.3.9",
         "@angular/language-service": "17.3.9",
+        "@playwright/test": "^1.44.1",
```

Comment:
Why do we need `playwright` dependency for the whole project?

Reply:
your right thanks, it got added accidentally while i was facing some test issues locally